unyt/_on_demand_imports.py

```diff
@@ -40,7 +59,7 @@ def __init_subclass__(cls):
 
     def __new__(cls):
         if cls is OnDemand:
-            raise TypeError("The OnDemand base class cannot be instantiated.")
```

Comment:
The old spelling is correct in American English, FWIW

Reply:
I thought it was. Well, it only got there because I copy-pasted the code from yt again.